packages/wallet/src/redux/actions.ts

```diff
@@ -53,6 +53,15 @@ export interface DisplayMessageSent {
   type: "WALLET.DISPLAY_MESSAGE_SENT";
 }
 
+export interface AssetTransferredEvent {
+  type: "WALLET.ASSET_HOLDER.ASSET_TRANSFERRED";
+
+  // This is either the `channelId` (bytes32) or (the left-padded bytes20)
+  // recipient address
```

Comment:
Nit: this should *always* be the left-padded recipient address (now known as an external destination). The event is intended to be emitted whenever [`_transferAsset`](https://github.com/statechannels/monorepo/blob/85ea52a16ebb36d024465c024fb3ecf91011aec0/packages/nitro-protocol/contracts/ETHAssetHolder.sol#L46) is called, and that function only accepts an `address` (20 bytes) for the destination. 

The dev-comments hopefully make it clear: 
https://github.com/statechannels/monorepo/blob/95c18a3d7de10d1a7d092d147bde0035dce8000d/packages/nitro-protocol/contracts/interfaces/IAssetHolder.sol#L43

Thinking about it, however, I suggest we change the event to emit the `address` (20 bytes) itself rather than the padded `bytes32`. This will require an additional call to `_bytes32ToAddress`, but in fact we will always have called that when preparing the call to ` _transferAsset`, so we can we can just keep the output in memory. This would hopefully remove any remaining ambiguity. Thoughts?

Reply:
Chatted offline about this: this will be a follow up PR.